docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -43,13 +43,16 @@ component "prometheus-operator" {
     }
   }
 
-  prometheus_metrics_retention = "14d"
```

Comment:
This changes should be merged with first commit, so block moving in Go code is done together with docs, as it should be committed as one unit.